tasks/StoringGroceries.tex

```diff
@@ -1,39 +1,44 @@
-\section{Storing Groceries [Housekeeper]}
+\section{Storing Groceries}
 \label{test:storing-groceries}
-The robot stores groceries into a pantry shelf while paying attention to sorting objects in their appropriate place, i.e.~storing an apple next to other fruits.
+The robot stores groceries into a pantry shelf while adhering to how objects are arranged in it, e.g., storing an apple next to other fruits.
+
+\noindent \textbf{Focus:} \SysI, \MAN, \CV, \OR.
 
 % \subsection*{Focus}
 % This test focuses on the detection and recognition of objects and their features, as well as object manipulation.
 
 \subsection*{Main Goal}
-Move objects 5 from a table into a shelf, grouping them by category or similarity.
+Move 5 objects from a table into a shelf, grouping them by category.
```

Comment:
Removing "or similarity" here makes it a bit inconsistent with "move an object next to similar objects in the shelf" in the scoresheet. For example, is it allowed to group unknown objects by size, shape, color, etc.?

Reply:
Yeah, grouping by size, shape, color is NOT ok. At least that is not how I usually arrange my pantry. That's why I changed it there. Will adjust the scoresheet as well.

Reply:
Adjusted